test/http.js

```diff
@@ -81,13 +81,13 @@ describe('HTTP APIs', function () {
   }
 
   describe('GET Root container', function () {
-    it('should have Access-Control-Allow-Origin as the req.Origin', function (done) {
-      server.get('/')
-        .set('Origin', 'http://example.com')
-        .expect('Access-Control-Allow-Origin', 'http://example.com')
-        .expect(200, done)
-    })
-
+    it('should have Access-Control-Allow-Origin as the req.Origin',
+      function (done) {
+        server.get('/')
+          .set('Origin', 'http://example.com')
+          .expect('Access-Control-Allow-Origin', 'http://example.com')
+          .expect(200, done)
+      })
     it('should exist', function (done) {
```

Comment:
Heh heh. They're all wrapped in `function (done)`. I just hard-wrapped the lines at 80 chars.
Given the fact that before, they were past 80 chars (on the same line as the `it('...', `), you couldn't easily see them on Github's diffs. Which is *precisely* why I hard-wrapped em :)